### DISCUSSION TITLE
Bridge Bluesky Safety account to fediverse

### DISCUSSION THREAD

**Discussion by zaphbbrox** [snarfed/bridgy-fed#1110] [Ideas]
Would be nice to have https://bsky.app/profile/safety.bsky.app in the fediverse also!
Or is it already there and I just wasn't able to find it?

**Comment by snarfed** [comment]
Agreed! I don't own that account, but feel free to petition them to follow https://bsky.app/profile/ap.brid.gy !

**Reply by zaphbbrox** [reply]
Okay, thought you're moderating it.
I'll write them a message!

**Reply by zaphbbrox** [reply]
Just realized, I overlooked Bluesky in the account name and automatically replaced it with Bridgy :D 
Makes sense, that you don't own that account!
